### ISSUE TITLE
right click -> New (File) context menu causing freezes all the time in WSL

### ISSUE THREAD

**Issue by mklueh** [iguissouma/nx-console-idea-plugin#133]
**Describe the bug:**
When this plugin is activated, doing right click in the IDE to create a new file will freeze the context menu for a while.

Likely the "Nx Generate (UI)" sub-menu is loading something and causing this issue.

**Steps to reproduce:**
It could be related to WSL 2, where my projects are while being developed with WebStorm on Windows, but a simple right click to create a new file causing this freeze. After waiting, the consequent right clicks are faster a lot, but occasionally it will be stuck again.

Currently it makes the development a bit of a pain, which is why I keep this plugin disabled for now.

**Comment by iguissouma** [comment]
@mklueh 
Yes first time it can be slow as it needs to calculate the schematics that are available on workspace(on recent plugin versions it should be loaded on background and until that the menu appear disabled)

Once loaded it's cached an no recalculated each time (the cache has a duration) or it's invalidated when other package or dependency changes...

Maybe will review this behaviour to not show children menu actions but a popup(the schematics will be loaded only when popup is showed).

**Comment by mklueh** [comment]
Hi, me again.

are there any news about this issue?

Not sure about others, but it really disturbes the workflow and is totally annoying having to deal with a freezing context menu.

**Comment by iguissouma** [comment]
Hi, 
I will try to take a look to this issue this week.
Sorry for the delay

**Comment by iguissouma** [comment]
@mklueh 
Can you see if this new version works better for you?
Install plugin from disk > choose the zip file.

[Nx Console Idea-0.58.0.zip](https://github.com/iguissouma/nx-console-idea-plugin/files/11242937/Nx.Console.Idea-0.58.0.zip)
